fe/fe-core/src/main/java/org/apache/doris/cloud/transaction/CloudGlobalTransactionMgr.java

```diff
@@ -355,6 +355,9 @@ public void commitTransaction(long dbId, List<Table> tableList, long transaction
                     tableList.stream().map(Table::getId).collect(Collectors.toList()));
             Map<Long, List<TCalcDeleteBitmapPartitionInfo>> backendToPartitionInfos = null;
             if (!mowTableList.isEmpty()) {
+                if (!checkTransactionStateBeforeCommit(dbId, transactionId)) {
```

Comment:
You should add a `checkTransactionStateBeforeCommit` check before all `commitTransactionWithoutLock` method calls

Reply:
@zhannngchen done